components/locale.js

```diff
@@ -61,14 +61,26 @@ function getInitialLocaleProps({req} = {}) {
   return Promise.resolve({locale})
 }
 
-// eslint-disable-next-line complexity
 function getContent(locale, options) {
+  const fallbackContent = content('', options)
+  if (locale === fallbackLocale) {
+    return fallbackContent
+  }
+  const localeContent = content(`${locale}/`, options)
+  return Object.keys(fallbackContent).reduce((cont, key) => {
```

Comment:
Oh snap. You're right. Hmmm... That's a real bummer. Hmmm... Let's just do shallow for now. We'll have to circle back once things have settled a bit and maybe change how we do things a bit...